apm-agent-plugins/apm-aws-sdk/apm-aws-sdk-1-plugin/src/main/java/co/elastic/apm/agent/awssdk/v1/helper/S3Helper.java

```diff
@@ -35,4 +37,36 @@ public static S3Helper getInstance() {
     public S3Helper(Tracer tracer) {
         super(tracer, SdkV1DataSource.getInstance());
     }
+
+    @Nullable
+    @Override
+    protected String getObjectKey(Request<?> request, @Nullable String bucketName) {
```

Comment:
[for reviewer] with SDK v1 we have to use some manual parsing of an HTTP header, this is not the case with v2.